var/spack/repos/builtin/packages/covfie/package.py

```diff
@@ -20,6 +20,7 @@ class Covfie(CMakePackage, CudaPackage):
 
     maintainers("stephenswat")
 
+    version("0.11.0", sha256="39fcd0f218d3b4f3aacc6af497a8cda8767511efae7a72b47781f10fd4340f4f")
```

Comment:
CMake minimum is now 3.21.

COVFIE_REQUIRE_CXX20 is removed and c++20 is now required, https://github.com/acts-project/covfie/compare/v0.10.0...v0.11.0#diff-5e713e1875f76db7b6b0a94d498a608b1b0ce4295abefc00287eb649e912e367. I think the `concepts` variant should be removed, or set to default=True now (removing a variant messes with configs that have `covfie +concepts` which will effectively remain pinned at 0.10.0).

Reply:
I just removed the variant all together. Given the very small userbase of this package I thought that would be fine.

Reply:
Removing variant altogether is fine. Did you want to update the cmake minimum?